src/services/exit-logs/exit-logs.spec.ts

```diff
@@ -0,0 +1,114 @@
+import dotenv from 'dotenv'
+
+import { ExitLogsService, makeExitLogsService } from './service.js'
+
+import {
+  LoggerService,
+  RequestService,
+  abort,
+  makeLogger,
+  makeRequest,
+  notOkError,
+  retry,
+} from '../../lib/index.js'
+
+import { mockConfig } from '../../test/config.js'
+import { ConfigService } from '../config/service.js'
+import { MetricsService } from 'services/prom/service.js'
+import {
+  ExecutionApiService,
+  makeExecutionApi,
+} from '../execution-api/service.js'
+
+dotenv.config()
+
+describe('exitLogs e2e', () => {
+  let api: ExitLogsService
+  let executionApi: ExecutionApiService
+  let request: RequestService
+  let logger: LoggerService
+  let config: ConfigService
+
+  const metrics = {
+    eventSecurityVerification: {
+      inc: vi.fn(),
+    },
+  } as unknown as MetricsService
+
+  const loadServices = () => {
+    executionApi = makeExecutionApi(request, logger, config)
+
+    api = makeExitLogsService(logger, executionApi, config, metrics)
+  }
+
+  beforeEach(async () => {
+    request = makeRequest([retry(3), notOkError(), abort(30_000)])
+    logger = makeLogger({
+      level: 'info',
+      format: 'simple',
+    })
+    config = mockConfig(logger, {
+      EXECUTION_NODE: 'https://eth.drpc.org',
+      CONSENSUS_NODE: 'https://ethereum-beacon-api.publicnode.com',
```

Comment:
I'd prefer better to store in GitHub vars. In case if this node will be unavailable, we replace it without any changes in code